quiz.js

```diff
@@ -37,9 +50,23 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+ name: 'Stark' }, { $push: { member: 'Ned Stark'}});
```

Comment:
'member' is probably not the best name for this property, since it will be holding multiple people; 'members' would be better.